### PR TITLE
docs: add mainnet to `README.md`

### PR DESCRIPTION
**Pull request by seolaoh**

### CODE REVIEW

**Review by Pangssu — APPROVED** [review]
nice

File: README.md [hunk 1]
```diff
@@ -1,16 +1,52 @@
...
 
-If you want to build Kroma, check out the [Protocol Specs](./specs/).
+Kroma offers 10x lower transactions fees compared to the Ethereum mainnet, with native Ethereum security and Ethereum
+equivalence. Our roadmap to transition from Optimistic rollup to ZK rollup sets Kroma apart from other L2 networks.
+Additionally, everyone in Kroma has the opportunity to participate as a validator. You can read our
+article '[The Road to Kroma's Decentralization](https://medium.com/@kroma-network/the-road-to-kromas-decentralization-38f8e46df442)'
+to learn more.
+
+Kroma is live on the mainnet on 09/05/23, and we're working on optimizing ZK Proofs to transition to ZK Rollup as part
+of our roadmap. also Kroma is the first rollup protocol to achieve **'Stage 2'**.
```
Comment (kangsorang) [inline README.md:23]:
```suggestion
of our roadmap. Also Kroma is the first rollup protocol to achieve **'Stage 2'**.
```

**Review by kangsorang — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,16 +1,52 @@
...
+Kroma aims to develop a universal ZK Rollup based on the Optimism Bedrock architecture. Currently, Kroma operates as an
+Optimistic Rollup with ZK fault proofs, utilizing a zkEVM based on Scroll. The goal of Kroma is to eventually transition
+to a ZK Rollup once the generation of ZK proofs becomes more cost-efficient and faster.
 
-If you want to build Kroma, check out the [Protocol Specs](./specs/).
+Kroma offers 10x lower transactions fees compared to the Ethereum mainnet, with native Ethereum security and Ethereum
+equivalence. Our roadmap to transition from Optimistic rollup to ZK rollup sets Kroma apart from other L2 networks.
+Additionally, everyone in Kroma has the opportunity to participate as a validator. You can read our
+article '[The Road to Kroma's Decentralization](https://medium.com/@kroma-network/the-road-to-kromas-decentralization-38f8e46df442)'
+to learn more.
```
Comment (0xHansLee) [inline README.md:20]:
```suggestion
Additionally, Kroma is advancing towards decentralization. As an initial step for this, Kroma has introduced a permissionless validator and challenge system. You can read our
article titled '[The Road to Kroma's Decentralization](https://medium.com/@kroma-network/the-road-to-kromas-decentralization-38f8e46df442)'
to learn more.
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,52 @@
...
+
+Kroma is live on the mainnet on 09/05/23, and we're working on optimizing ZK Proofs to transition to ZK Rollup as part
+of our roadmap. Also Kroma is the first rollup protocol to achieve **'Stage 2'**.
```
Comment (0xHansLee) [inline README.md:23]: I think this is not necessary here. But let's add the word `the first universal state 2 rollup` at the start of this description.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,52 @@
...
+  <br />
+  <h3><a href="https://kroma.network">Kroma</a> aims to be a New Universal ZK Rollup on Ethereum.</h3>
+  <br />
+</div>
 
-This is a client for Kroma network.
+## About Kroma
 
-## Documentation
+Kroma aims to develop a universal ZK Rollup based on the Optimism Bedrock architecture. Currently, Kroma operates as an
```
Comment (0xHansLee) [inline README.md:12]:
```suggestion
Kroma is the first stage 2 universal Layer 2 on Ethereum. Kroma has been developed based on Optimism Bedrock architecture, and aims to a universal ZK Rollup. Currently, Kroma operates as an
```

File: README.md [hunk 5]
```diff
@@ -1,16 +1,52 @@
...
+equivalence. Our roadmap to transition from Optimistic rollup to ZK rollup sets Kroma apart from other L2 networks.
+Additionally, everyone in Kroma has the opportunity to participate as a validator. You can read our
+article '[The Road to Kroma's Decentralization](https://medium.com/@kroma-network/the-road-to-kromas-decentralization-38f8e46df442)'
+to learn more.
+
+Kroma is live on the mainnet on 09/05/23, and we're working on optimizing ZK Proofs to transition to ZK Rollup as part
+of our roadmap. Also Kroma is the first rollup protocol to achieve **'Stage 2'**.
+
+For more detailed information about Kroma, check [Kroma docs](https://docs.kroma.network) (or
+[Markdown docs source](./specs)).
```
Comment (0xHansLee) [inline README.md:26]:
```suggestion
[specs](./specs)).
```

File: README.md [hunk 6]
```diff
@@ -1,16 +1,52 @@
...
+
+## Links
+
+- Official Website: [Kroma](https://kroma.network)
+- Documentation: [Kroma Docs](https://docs.kroma.network)
+- Blog: [Kroma Medium](https://medium.com/@kroma-network)
+- Discord: [Kroma](https://discord.gg/kroma)
```
Comment (0xHansLee) [inline README.md:33]:
```suggestion
- Discord: [Kroma discord](https://discord.gg/kroma)
```

File: README.md [hunk 7]
```diff
@@ -1,16 +1,52 @@
...
+- X: [Kroma](https://twitter.com/kroma_network)
```
Comment (0xHansLee) [inline README.md:34]:
```suggestion
- X: [Kroma X](https://twitter.com/kroma_network)
```

**Review by 0xHansLee — COMMENTED** [review]
Thank you for writing this. I left my opinions. Please check this.

**Review by 0xHansLee — APPROVED** [review]
LGTM!

**Pull request merged**